gm4_trash_cans/data/gm4_trash_cans/functions/update.mcfunction

```diff
@@ -0,0 +1,11 @@
+# @s = trash_can
+# run from main
+
+# Checks if trash can has items inside it
+function gm4_trash_cans:check_dropper
```

Comment:
```suggestion
execute if data block ~ ~ ~ Items[0] run function gm4_trash_cans:check_dropper
```

Reply:
If this is done here and not in the function after. Then none of the particle effects or visuals update properly. Not sure if it breaks functionality yet.